### DISCUSSION TITLE
Remotely docker deployment error

### DISCUSSION THREAD

**Discussion by goftari** [immense/Remotely#557] [Q&A]
Server: Centos 7.9.2009 (Core)
Docker version:  1.13.1, build 7d71120/1.13.1
I get the following error when trying to the latest version of immybot/remotely:latest docker image:
`Entered main script.
Starting Remotely server.
Failed to create CoreCLR, HRESULT: 0x80070008`

**Comment by newrivertech** [comment]
From https://docs.linuxserver.io/faq,

> Option 1 (Long-Term Fix)
> Upgrade your Docker engine install to at least version 20.10.10. 
> ​
> Option 2 (Short-Term Fix)
> For Docker CLI, run your container with:
> --security-opt seccomp=unconfined
> For Docker Compose, run your container with:
> `security_opt:
>     - seccomp=unconfined`
